--- main.go	2022-04-20 16:32:38.283785115 +0200
+++ main-change.go	2022-04-20 16:34:58.502292974 +0200
@@ -241,11 +241,6 @@
 	out.Metadata.DefaultRepositoryPermission = &drp
 	out.Metadata.MembersCanCreateRepositories = &meta.MembersCanCreateRepositories
 
-	var runningAsAdmin bool
-	runningAs, err := client.BotUser()
-	if err != nil {
-		return nil, fmt.Errorf("failed to obtain username for this token")
-	}
 	admins, err := client.ListOrgMembers(orgName, github.RoleAdmin)
 	if err != nil {
 		return nil, fmt.Errorf("failed to list org admins: %w", err)
@@ -254,13 +249,6 @@
 	for _, m := range admins {
 		logrus.WithField("login", m.Login).Debug("Recording admin.")
 		out.Admins = append(out.Admins, m.Login)
-		if runningAs.Login == m.Login {
-			runningAsAdmin = true
-		}
-	}
-
-	if !runningAsAdmin {
-		return nil, fmt.Errorf("--dump must be run with admin:org scope token")
 	}
 
 	orgMembers, err := client.ListOrgMembers(orgName, github.RoleMember)
